smoketests/__init__.py

```diff
@@ -143,7 +143,7 @@ def reset_config(cls):
 
     def fingerprint(self):
         # Fetch the server's fingerprint; required for `identity list`.
-        self.spacetime("server", "fingerprint", "localhost", "-f")
+        self.spacetime("server", "fingerprint", "-s", "localhost", "-f")
```

Comment:
the local server is called `local` not `localhost` I believe

Reply:
also we've changed this recently so it makes sense this is failing

Reply:
it looks like we explicitly configure the server in `smoketests/config.toml`

Reply:
No, that's not the issue - it's passing now.